File: Bridge/Resources/.generated/system/iO/invalidDataException.js
    Bridge.define("System.IO.InvalidDataException", {
        inherits: [System.SystemException],
        ctors: {
            ctor: function () {
                this.$initialize();
                System.SystemException.ctor.call(this, "Invalid data format.");
            },
            $ctor1: function (message) {
                this.$initialize();
                System.SystemException.ctor.call(this, message);
            },
            $ctor2: function (message, innerException) {
                this.$initialize();
                System.SystemException.ctor.call(this, message, innerException);
            }
        }
    });

File: Bridge/System/IO/InvalidDataException.cs
using Bridge;

namespace System.IO
{
    public sealed class InvalidDataException : SystemException
    {
        public InvalidDataException()
            : base("Invalid data format.")
        {
        }

        public InvalidDataException(string message)
            : base(message)
        {
        }

        public InvalidDataException(string message, Exception innerException)
            : base(message, innerException)
        {
        }
    }
}